downstream/modules/platform/ref-ports-firewalls-requirements.adoc

```diff
@@ -35,54 +37,61 @@ ALLOW connections from controller(s) to database port
 .{ControllerNameStart}
 [options="header"]
 |===
-|Port |Protocol |Service |Installer Inventory Variable |Required for
+|Port |Protocol |Service |Direction |Installer Inventory Variable |Required for
 |22
 |TCP
 |SSH
-|ansible_port
+|Bi-directional
+|`ansible_port`
 |Installation
 
 |80
 |TCP
 |HTTP
-|nginx_http_port
+|Incoming
+|`nginx_http_port`
 |UI/API
 
 |443
 |TCP
 |HTTPS
-|nginx_https_port
+|Incoming
+|`nginx_https_port`
 |UI/API
 
 |5432
 |TCP
 |PostgreSQL
-|pg_port
+|Bi-directional
+|`pg_port`
 a|Open *only* if the internal database is being used along with another component. Otherwise, this port should not be opened
 
 Hybrid mode in a cluster
 
 |27199
 |TCP
 |Receptor
-|receptor_listener_port
+|Bi-directional
+|`receptor_listener_port`
 |ALLOW receptor port across all controllers (for mandatory & automatic control plane peering)
 |===
 
 .Hop Nodes
 [options="header"]
 |===
-|Port |Protocol |Service |Installer Inventory Variable |Required for
+|Port |Protocol |Service |Direction |Installer Inventory Variable |Required for
 |22
 |TCP
 |SSH
-|ansible_port
+|Bi-directional
+|`ansible_port`
 |Installation
 
 |27199
 |TCP
 |Receptor
-|receptor_listener_port
+|Bi-directional
+|`receptor_listener_port`
 a|Mesh
 
 ALLOW connection from controller(s) to receptor port
```

Comment:
```suggestion
ALLOW connection from controller(s) to Receptor nodes.
```

Reply:
There are a few more instances of receptor (lowercase) that I think should be Receptor. 

https://github.com/ansible/receptor

Reply:
I was told that a Receptor node and a Receptor port are two different things. I am going with what the SME told me to use.

Reply:
Yes, a node and a port are two different things. You connect to a node through a port.